python/tests/res/simulator/test_simulation_context.py

```diff
@@ -29,38 +27,28 @@ def test_simulation_context(self):
                 mask2[2*iens_2] = False
                 mask2[2*iens_2 + 1] = True
 
-
             fs_manager = ert.getEnkfFsManager()
             first_half = fs_manager.getFileSystem("first_half")
             other_half = fs_manager.getFileSystem("other_half")
 
-            simulation_context1 = SimulationContext(ert, first_half, mask1 , 0)
-            simulation_context2 = SimulationContext(ert, other_half, mask2 , 0)
-
-            ert.createRunpath( simulation_context1.get_run_context( ) )
-            ert.createRunpath( simulation_context2.get_run_context( ) )
+            # i represents geo_id
+            case_data = [(i,0) for i in range(size)]
```

Comment:
This is wrong usage of the API. Please revise the documentation: https://github.com/equinor/libres/blob/9ba9839785764568e6479b34005154bfb8b7575f/python/res/simulator/batch_simulator.py#L133

The following should be correct:
```python
case_data = [(i, {}) for i in range(size)]
```

Reply:
ok, I knew it was supposed to be a dict, but since it not used anyway ... fixed now